packages/react-aria-components/src/Table.tsx

```diff
@@ -1154,6 +1176,8 @@ function TableCell<T>({cell}: {cell: GridNode<T>}) {
       {...mergeProps(filterDOMProps(props as any), gridCellProps, focusProps, hoverProps)}
       {...renderProps}
       ref={ref}
+      data-hovered={isHovered || undefined}
+      data-interactive={isInteractive || undefined}
```

Comment:
Do you think we need this on Cell if it's really reflecting properties of the Row? people could do `.react-aria-Row[data-interactive] .react-aria-Cell` in CSS to get this. perhaps we omit it for now and see if it causes confusion?